Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -1772,7 +1773,9 @@ void addResult(HashMap<WebCore::FrameIdentifier, Ref<WebCore::LegacyWebArchive>>
         @"axIsEnabled": [NSNumber numberWithBool:result.isAccessibilityEnabled],
         @"axIsThreadInitialized": [NSNumber numberWithBool:result.isAccessibilityThreadInitialized],
         @"axLiveTree": result.liveTree.createNSString().get(),
-        @"axIsolatedTree": result.isolatedTree.createNSString().get()
+        @"axIsolatedTree": result.isolatedTree.createNSString().get(),
+        @"axRemoteTokenHash": [NSNumber numberWithUnsignedInteger:result.remoteTokenHash],
```

Comment:
Both tokens are "remote" depending on your perspective. What if we call this one axUIProcessGeneratedTokenHash?

Or alternatively, if we're going to display all four hashes, maybe we want:

axUIProcessLocalHash
axUIProcessRemoteHash
axWebProcessLocalHash
axWebProcessRemoteHash

where we expect axUIProcessLocalHash == axWebProcessRemoteHash, etc.